canonical-web-and-design/canonicalwebteam.search
# Add optional site restricted search

## Done
This change reverts the previous hard change to `/siterestricted` and allows renovate PRs to be unblocked, so that search works even without providing a new search engine id

## QA
- Main site to test is u.com because providing an explicit "switch" means we are adding an additional argument that didn't exist https://github.com/canonical-web-and-design/ubuntu.com/pull/11471
- Other sites should not require any code change for this update e.g. https://github.com/canonical-web-and-design/snapcraft.io/pull/3952


Fixes https://github.com/canonical-web-and-design/web-squad/issues/5086
